test/plugin/test_in_http.rb

```diff
@@ -517,6 +517,28 @@ def test_application_json
     assert_equal_event_time time, d.events[1][1]
   end
 
+  def test_csp_report
+    d = create_driver
+    time = event_time("2011-01-02 13:14:15 UTC")
+    time_i = time.to_i
+    events = [
+      ["tag1", time, {"a"=>1}],
+      ["tag2", time, {"a"=>2}],
+    ]
```

Comment:
It would be better that the sample contents contains actual syntax of CSP report:

* [report syntax](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only#violation_report_syntax)
* [Sample violation report](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only#sample_violation_report)

Reply:
It's a trivial thing, no need to block merging.